src/main/features/ccj/views/redetermination.njk

```diff
@@ -1,19 +1,33 @@
 {% extends "layout.njk" %}
 
+{% if madeBy === 'CLAIMANT'%}
 {% set heading = 'Why do you believe the defendant can repay you sooner?' %}
+{% else %}
+{% set heading = 'Ask a judge to make a repayment plan' %}
+{% endif %}
 
-{% from "form.njk" import saveAndContinueButton, csrfProtection, errorSummary, textArea %}
+{% from "form.njk" import saveAndContinueButton, submitButton, csrfProtection, errorSummary, textArea %}
+{% from "repaymentPlanBlock.njk" import repaymentPlanBlock %}
 
 {% block content %}
-  <div class="column-two-thirds">
+<div class="column-two-thirds">
 
-    <form method="post" >
-      {{ csrfProtection(csrf) }}
+  <form method="post">
+    {{ csrfProtection(csrf) }}
 
-      {{ textArea(name = 'text', form = form, rows = 5, columns=18) }}
+    {% if madeBy === 'DEFENDANT'%}
+    {{ repaymentPlanBlock( heading = 'The claimant has proposed this repayment plan', repaymentPlan = repaymentPlan ) }}
+    {% endif %}
 
-      {{ saveAndContinueButton() }}
-    </form>
+    {{ textArea(label = 'Why do you feel this plan is not affordable?', name = 'text', form = form, rows = 5,
```

Comment:
I think this label would be different for `CLAMANT`, The one you are using here is for `DEFENDANT`.